### PR TITLE
Add initial Audit Manager Control Tower benchmarks and controls. Closes #315

### LINKED ISSUE #315: Add initial Audit Manager Control Tower benchmarks and controls
**Is your feature request related to a problem? Please describe.**
Add Audit Manager Control Tower benchmarks and controls

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
https://docs.aws.amazon.com/audit-manager/latest/userguide/controltower.html

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: control_tower/control_tower.sp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+benchmark "audit_manager_control_tower" {
+  title         = "AWS Audit Manager Control Tower Guardrails"
+  description   = "AWS Control Tower is a service that enables you to enforce and manage governance rules for security, operations, and compliance at scale across all your organizations and accounts in the AWS Cloud."
+  documentation = file("./control_tower/docs/control_tower_overview.md")
+  children = [
+    benchmark.audit_manager_control_tower_ebs_checks,
+    benchmark.audit_manager_control_tower_disallow_internet_connection,
+    benchmark.audit_manager_control_tower_multi_factor_authentication,
+    benchmark.audit_manager_control_tower_disallow_public_access,
+    benchmark.audit_manager_control_tower_disallow_instances,
```
Comment (misraved) [inline control_tower/control_tower.sp:17]:
```suggestion
    benchmark.audit_manager_control_tower_disallow_instances
```

File: control_tower/disallow_instances.sp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    control_set = "disallow_instances"
+  })
+}
+
+benchmark "audit_manager_control_tower_disallow_instances" {
+  title        = "Disallow Instances"
+  description  = "This benchmark checks if RDS storage are encrypted and S3 buckets versioning is enabled."
+  children = [
+    benchmark.audit_manager_control_tower_disallow_instances_5_0_1,
+    benchmark.audit_manager_control_tower_disallow_instances_5_1_1,
```
Comment (misraved) [inline control_tower/disallow_instances.sp:12]:
```suggestion
    benchmark.audit_manager_control_tower_disallow_instances_5_1_1
```

File: control_tower/disallow_instances.sp [hunk 3]
```diff
@@ -0,0 +1,39 @@
+locals {
+  audit_manager_control_tower_disallow_instances_common_tags = merge(local.audit_manager_control_tower_common_tags, {
+    control_set = "disallow_instances"
+  })
+}
+
+benchmark "audit_manager_control_tower_disallow_instances" {
+  title        = "Disallow Instances"
+  description  = "This benchmark checks if RDS storage are encrypted and S3 buckets versioning is enabled."
```
Comment (misraved) [inline control_tower/disallow_instances.sp:9]:
```suggestion
  description  = "This benchmark checks if RDS storage is encrypted and S3 bucket's versioning is enabled."
```

File: control_tower/disallow_instances.sp [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  ]
+
+  tags = merge(local.audit_manager_control_tower_disallow_instances_common_tags, {
+    audit_manager_control_tower_item_id  = "5.0.1"
+  })
+}
+
+benchmark "audit_manager_control_tower_disallow_instances_5_1_1" {
+  title         = "5.1.1 - Disallow S3 buckets that are not versioning enabled"
+  description   = "Disallow S3 buckets that are not versioning enabled - Checks whether versioning is enabled for your S3 buckets."
```
Comment (misraved) [inline control_tower/disallow_instances.sp:31]:
```suggestion
  description   = "Disallow S3 buckets that do not have versioning enabled - Checks whether versioning is enabled for your S3 buckets."
```

File: control_tower/disallow_internet_connection.sp [hunk 5]
```diff
@@ -0,0 +1,39 @@
+locals {
+  audit_manager_control_tower_disallow_internet_connection_common_tags = merge(local.audit_manager_control_tower_common_tags, {
+    control_set = "disallow_internet_connection"
+  })
+}
+
+benchmark "audit_manager_control_tower_disallow_internet_connection" {
+  title         = "Disallow Internet Connection"
+  description   = "This benchmark checks if VPC security group restrict ingress from RDP and SSH."
```
Comment (misraved) [inline control_tower/disallow_internet_connection.sp:9]:
```suggestion
  description   = "This benchmark checks if the VPC security group restricts ingress from RDP and SSH."
```

File: control_tower/disallow_public_access.sp [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    benchmark.audit_manager_control_tower_disallow_public_access_4_0_2,
+    benchmark.audit_manager_control_tower_disallow_public_access_4_1_1,
+    benchmark.audit_manager_control_tower_disallow_public_access_4_1_2
+  ]
+  tags          = local.audit_manager_control_tower_disallow_public_access_common_tags
+}
+
+benchmark "audit_manager_control_tower_disallow_public_access_4_0_1" {
+  title         = "4.0.1 - Disallow public access to RDS database instances"
+  description   = "Disallow public access to RDS database instances - Checks whether the Amazon Relational Database Service (RDS) instances are not publicly accessible. The rule is non-compliant if the publiclyAccessible field is true in the instance configuration item."
```
Comment (misraved) [inline control_tower/disallow_public_access.sp:21]:
```suggestion
  description   = "Disallow public access to RDS database instances - Checks whether the Amazon Relational Database Service (RDS) instances are publicly accessible. The rule is non-compliant if the `publiclyAccessible` field is true in the instance configuration item."
```

File: control_tower/ebs_checks.sp [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  })
+}
+
+benchmark "audit_manager_control_tower_ebs_checks" {
+  title         = "EBS checks"
+  description   = "This benchmark checks if EBS volumes are in use, encrypted etc."
+  children = [
+    benchmark.audit_manager_control_tower_ebs_checks_1_0_1,
+    benchmark.audit_manager_control_tower_ebs_checks_1_0_2,
+    benchmark.audit_manager_control_tower_ebs_checks_1_0_3,
```
Comment (misraved) [inline control_tower/ebs_checks.sp:13]:
```suggestion
    benchmark.audit_manager_control_tower_ebs_checks_1_0_3
```

File: control_tower/multi_factor_authentication.sp [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+  })
+}
+
+benchmark "audit_manager_control_tower_multi_factor_authentication" {
+  title         = "Multi-Factor Authentication"
+  description   = "This benchmark checks if MFA is enabled for all the IAM users."
+  children = [
+    benchmark.audit_manager_control_tower_multi_factor_authentication_3_0_1,
+    benchmark.audit_manager_control_tower_multi_factor_authentication_3_0_2,
+    benchmark.audit_manager_control_tower_multi_factor_authentication_3_0_3,
```
Comment (misraved) [inline control_tower/multi_factor_authentication.sp:13]:
```suggestion
    benchmark.audit_manager_control_tower_multi_factor_authentication_3_0_3
```

**Review by misraved — COMMENTED** [review]
1. Can we update `control_tower = true` tag to `audit_manager_control_tower = true`
2. Make changes as per the review comments

**Review by misraved — APPROVED** [review]

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**